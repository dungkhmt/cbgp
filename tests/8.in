50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0