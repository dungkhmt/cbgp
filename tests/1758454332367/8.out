(14, 3), (49, 42), (36, 15), (19, 11), (27, 6), (47, 17), (47, 2), (21, 5), (28, 7), (34, 29), (32, 26), (37, 4), (20, 5), (29, 14), (4, 5), (32, 13), (43, 0), (6, 22), (49, 36), (18, 20), (43, 10), (32, 10), (6, 2), (24, 0), (41, 6), (28, 1), (36, 41), (48, 24), (32, 25), (45, 4), (30, 9), (20, 23), (38, 6), (24, 7), (35, 45), (49, 39), (9, 4), (9, 12), (15, 0), (25, 9), (5, 8), (26, 11), (3, 0), (40, 15), (34, 16), (46, 39), (44, 26), (32, 33), (31, 9), (47, 7), (37, 45), (27, 28), (32, 11), (29, 3), (24, 2), (2, 3), (35, 37), (34, 14), (32, 9), (27, 1), (22, 14), (3, 16), (13, 10), (42, 18), (18, 5), (17, 7), (42, 5), (11, 16), (42, 15), (12, 0), (43, 19), (30, 23), (27, 36), (36, 1), (26, 4), (32, 31), (11, 3), (43, 33), (18, 23), (6, 3), 
numberIntersection=0 
minDistance=2.23606797749979 
minAngle=0.042080407441890344 
sumAngle=57.23118666260629 
sumDistance=254520.94480246145 
minEdgeLength=2.23606797749979 
sumEdgeLength=2476.4400892601197 

0.0000000000 2.2360679775 0.0420804074 57.2311866626 254520.9448024615 2.2360679775 2476.4400892601
0: (57, 23),
1: (140, 3),
2: (110, 23),
3: (107, 25),
4: (52, 84),
5: (1, 4),
6: (115, 56),
7: (135, 3),
8: (0, 200),
9: (69, 61),
10: (67, 37),
11: (96, 28),
12: (61, 36),
13: (70, 34),
14: (105, 56),
15: (50, 4),
16: (100, 30),
17: (131, 6),
18: (43, 17),
19: (68, 27),
20: (18, 17),
21: (0, 1),
22: (112, 56),
23: (16, 23),
24: (82, 6),
25: (76, 47),
26: (94, 94),
27: (137, 11),
28: (137, 4),
29: (103, 34),
30: (18, 25),
31: (79, 51),
32: (88, 34),
33: (71, 31),
34: (100, 34),
35: (42, 139),
36: (145, 0),
37: (46, 85),
38: (200, 200),
39: (5, 0),
40: (54, 7),
41: (132, 27),
42: (43, 4),
43: (66, 30),
44: (197, 200),
45: (43, 183),
46: (2, 0),
47: (132, 9),
48: (56, 6),
49: (43, 0),
57 23 140 3 110 23 107 25 52 84 1 4 115 56 135 3 0 200 69 61 67 37 96 28 61 36 70 34 105 56 50 4 100 30 131 6 43 17 68 27 18 17 0 1 112 56 16 23 82 6 76 47 94 94 137 11 137 4 103 34 18 25 79 51 88 34 71 31 100 34 42 139 145 0 46 85 200 200 5 0 54 7 132 27 43 4 66 30 197 200 43 183 2 0 132 9 56 6 43 0 
0: (57, 23), 1: (140, 3), 2: (110, 23), 3: (107, 25), 4: (52, 84), 5: (1, 4), 6: (115, 56), 7: (135, 3), 8: (0, 200), 9: (69, 61), 10: (67, 37), 11: (96, 28), 12: (61, 36), 13: (70, 34), 14: (105, 56), 15: (50, 4), 16: (100, 30), 17: (131, 6), 18: (43, 17), 19: (68, 27), 20: (18, 17), 21: (0, 1), 22: (112, 56), 23: (16, 23), 24: (82, 6), 25: (76, 47), 26: (94, 94), 27: (137, 11), 28: (137, 4), 29: (103, 34), 30: (18, 25), 31: (79, 51), 32: (88, 34), 33: (71, 31), 34: (100, 34), 35: (42, 139), 36: (145, 0), 37: (46, 85), 38: (200, 200), 39: (5, 0), 40: (54, 7), 41: (132, 27), 42: (43, 4), 43: (66, 30), 44: (197, 200), 45: (43, 183), 46: (2, 0), 47: (132, 9), 48: (56, 6), 49: (43, 0), 
numberIntersection=0 minAngle=0.042080407441890344 sumAngle=57.231186662598596 minEdgeLength=2.23606797749979 sumEdgeLength=2476.4400892601225 minDistance=2.23606797749979 sumDistance=254520.94480246495 
Time taken: 13.845 s
